documentation/source/simulator_support.rst

```diff
@@ -121,6 +121,19 @@ To get waveforms in VCD format, add Verilator's trace option(s) to the
 To set the same options on the command line, use ``EXTRA_ARGS="--trace --trace-structs" make ...``.
 A VCD file named ``dump.vcd`` will be generated in the current directory.
 
+Verilator can produce waveform traces in the FST format, the native format of GTKWave. 
+FST traces are much smaller and more efficient to write, but require the use of GTKWave.
+
+To enable FST tracing, add `--trace-fst -CFLAGS -DVM_TRACE_FST=1` to `EXTRA_ARGS` as shown below.
+For Verilator 4.102 and above, the `-CFLAGS -DVM_TRACE_FST=1` argument is no longer necessary.
+
+  .. code-block:: make
+
+    EXTRA_ARGS += --trace-fst --trace-structs -CFLAGS -DVM_TRACE_FST
+
+The resulting file will be ``dump.fst`` and can be opened by ``gtkwave dump.fst``.
+Note only the most recent versions of gtkwave and verilator support the FST format.
```

Comment:
GtkWave has been supporting FST for ages (ever?), and Verilator supported it also for multiple years (May 2018 actually, see https://github.com/verilator/verilator-announce/issues/17). Just remove this sentence. Earlier versions don't work with cocotb anyways.

Reply:
That is not completely accurate as I've experienced myself. There have been relatively recent updates (meaning this year) which have fixed a few issues that allowed it to be used. Before that performance was abysmal.

But I do not care, we can remove it, I just thought it might be a helpful hint.